server/integration/testsuite/src/test/java/org/infinispan/persistence/cluster/MyCustomCacheStore.java

```diff
@@ -8,6 +8,10 @@
 import org.infinispan.util.logging.Log;
 import org.infinispan.util.logging.LogFactory;
 
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.fail;
```

Comment:
Sorry I forgot this is in the integration test suite module which uses junit, so junit.Assert is fine :)